win32/Lib/winnt.py

```diff
@@ -161,34 +161,22 @@
 
 
 def PRIMARYLANGID(lgid):
-    return (WORD)(lgid) & 1023
+    return (lgid) & 1023
 
 
 def SUBLANGID(lgid):
-    return (WORD)(lgid) >> 10
+    return (lgid) >> 10
 
 
 NLS_VALID_LOCALE_MASK = 1048575
 
 
 def LANGIDFROMLCID(lcid):
-    return (WORD)(lcid)
```

Comment:
`(WORD)` and `(DWORD)` seem like invalid leftovers from C casting transpilation.

`SORTIDFROMLCID` and `LANGIDFROMLCID` are not referenced anywhere.